tests/functional/aws-node-sdk/test/bucket/deleteWebsite.js

```diff
@@ -0,0 +1,69 @@
+import assert from 'assert';
+
+import withV4 from '../support/withV4';
+import BucketUtility from '../../lib/utility/bucket-util';
+import { WebsiteConfigTester } from '../../lib/utility/website-util';
+
+const bucketName = 'testdeletewebsitebucket';
+
+describe('DELETE bucket website', () => {
+    withV4(sigCfg => {
+        const bucketUtil = new BucketUtility('default', sigCfg);
+        const s3 = bucketUtil.s3;
+        const otherAccountBucketUtility = new BucketUtility('lisa', {});
+        const otherAccountS3 = otherAccountBucketUtility.s3;
+
+        describe('without existing bucket', () => {
+            it('should return NoSuchBucket', done => {
+                s3.deleteBucketWebsite({ Bucket: bucketName }, err => {
+                    assert(err);
+                    assert.strictEqual(err.code, 'NoSuchBucket');
+                    assert.strictEqual(err.statusCode, 404);
+                    done();
+                });
+            });
+        });
+
+        describe('with existing bucket', () => {
+            beforeEach(() => s3.createBucketAsync({ Bucket: bucketName }));
+            afterEach(() => bucketUtil.deleteOne(bucketName));
+
+            describe('without existing configuration', () => {
+                it('should return a 200 response', done => {
+                    s3.deleteBucketWebsite({ Bucket: bucketName }, err => {
+                        assert.strictEqual(err, null,
+                            `Found unexpected err ${err}`);
+                        done();
+                    });
+                });
+            });
+
+            describe('with existing configuration', () => {
+                beforeEach(done => {
+                    const config = new WebsiteConfigTester('index.html');
+                    s3.putBucketWebsite({ Bucket: bucketName,
+                    WebsiteConfiguration: config }, done);
+                });
+
+                it('should delete bucket configuration successfully', done => {
+                    s3.deleteBucketWebsite({ Bucket: bucketName }, err => {
+                        assert.strictEqual(err, null,
+                            `Found unexpected err ${err}`);
+                        return done();
```

Comment:
`done` is returned sometimes, but not others. It suppose it is a good practice to return it consistently, though it's not always necessary.